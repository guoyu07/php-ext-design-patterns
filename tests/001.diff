001+ Segmentation fault
001- php_design_patterns extension is available
002+ 
003+ Termsig=11