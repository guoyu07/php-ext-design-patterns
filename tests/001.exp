php_design_patterns extension is available