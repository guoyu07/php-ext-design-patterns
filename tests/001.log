
---- EXPECTED OUTPUT
php_design_patterns extension is available
---- ACTUAL OUTPUT
Segmentation fault

Termsig=11
---- FAILED
