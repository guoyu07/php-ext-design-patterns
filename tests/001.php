<?php 
echo "php_design_patterns extension is available";
?>
