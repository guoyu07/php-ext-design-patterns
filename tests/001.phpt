--TEST--
Check for php_design_patterns presence
--SKIPIF--
<?php if (!extension_loaded("php_design_patterns")) print "skip"; ?>
--FILE--
<?php 
use PHPDesign\MacFactory;
use PHPDesign\WinFactory;
$macObj = new MacFactory();
$macObj->CreateButton();
$winObj = new WinFactory();
$winObj->CreateButton();
?>
--EXPECT--
MacFactory: CreateButton
WINFactory: CreateButton
