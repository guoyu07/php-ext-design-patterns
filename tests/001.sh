#!/bin/sh

/usr/local/php7master/bin/php  -n -c '/home/laputa/桌面/php-src-master/ext/php_design_patterns/tmp-php.ini'  -d "output_handler=" -d "open_basedir=" -d "disable_functions=" -d "output_buffering=Off" -d "error_reporting=32767" -d "display_errors=1" -d "display_startup_errors=1" -d "log_errors=0" -d "html_errors=0" -d "track_errors=0" -d "report_memleaks=1" -d "report_zend_debug=0" -d "docref_root=" -d "docref_ext=.html" -d "error_prepend_string=" -d "error_append_string=" -d "auto_prepend_file=" -d "auto_append_file=" -d "ignore_repeated_errors=0" -d "precision=14" -d "memory_limit=128M" -d "log_errors_max_len=0" -d "opcache.fast_shutdown=0" -d "opcache.file_update_protection=0" -d "extension_dir=/home/laputa/桌面/php-src-master/ext/php_design_patterns/modules/" -d "extension=php_design_patterns.so" -f "/home/laputa/桌面/php-src-master/ext/php_design_patterns/tests/001.php"  2>&1
