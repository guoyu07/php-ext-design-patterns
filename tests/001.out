Segmentation fault

Termsig=11